application/actions/entitycodes_test.go

```diff
@@ -67,11 +67,9 @@ func (as *ActionSuite) Test_entityCodesList() {
 
```

Comment:
Would still be good to check for this.

```suggestion
			as.NotNil(codes[0].Active)
```